.github/workflows/ci.yml

```diff
@@ -40,16 +40,14 @@ jobs:
     strategy:
       matrix:
         os:
-          - 'debian-10'
           - 'centos-7'
           - 'centos-8'
-          - 'ubuntu-2004'
+          - 'fedora-31'
```

Comment:
use `fedora-latest` to make your life easier :+1:

Reply:
I've used it in the past but decided to switch away from it so I could know which version was being testing with. It's a bit of a toss-up with a 6 month release cycle though as you point out.